0-bootstrap/outputs.tf

```diff
@@ -56,6 +56,9 @@ output "common_config" {
     parent_folder         = var.parent_folder,
     billing_account       = var.billing_account,
     default_region        = var.default_region,
+    default_region_2      = var.default_region_2,
+    default_region_gcs    = var.default_region_gcs,
+    default_region_kms    = var.default_region_kms,
```

Comment:
New variables in 0-bootstrap need to be added to the deploy helper

- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/helpers/foundation-deployer/global.tfvars.example#L31
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/helpers/foundation-deployer/stages/data.go#L131
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/helpers/foundation-deployer/stages/data.go#L193
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/helpers/foundation-deployer/stages/apply.go#L34

Reply:
Thanks added in cc14de8a4d7b8d89ca581cde46d08bd47bf84f99